KitePrintSDK/src/main/java/ly/kite/print/AssetUploadRequestListener.java

```diff
@@ -7,6 +9,6 @@
  */
 interface AssetUploadRequestListener {
     void onProgress(AssetUploadRequest req, int totalAssetsUploaded, int totalAssetsToUpload, long bytesWritten, long totalAssetBytesWritten, long totalAssetBytesExpectedToWrite);
-    void onUploadComplete(AssetUploadRequest req, List<Asset> assets);
+    void onUploadComplete(Context context, AssetUploadRequest req, List<Asset> assets);
```

Comment:
No - I think that was one of the hacks I needed to introduce quickly in 
order to resolve a problem elsewhere, but I think I've now done it a 
different way, so the context doesn't need to be passed out any more. 
I'll create a card for it.

On 21/07/15 16:45, Deon Botha wrote:
>
> In 
> KitePrintSDK/src/main/java/ly/kite/print/AssetUploadRequestListener.java 
> <https://github.com/OceanLabs/Android-Print-SDK/pull/9#discussion_r35117881>:
>
> > @@ -7,6 +9,6 @@
> >   */
> >  interface AssetUploadRequestListener {
> >      void onProgress(AssetUploadRequest req, int totalAssetsUploaded, int totalAssetsToUpload, long bytesWritten, long totalAssetBytesWritten, long totalAssetBytesExpectedToWrite);
> > -    void onUploadComplete(AssetUploadRequest req, List<Asset> assets);
> > +    void onUploadComplete(Context context, AssetUploadRequest req, List<Asset> assets);
>
> I'm curious, do we need to pass out the context here? Feels wrong from 
> a public interface point of view.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OceanLabs/Android-Print-SDK/pull/9/files#r35117881>.
>